sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -165,7 +162,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake { nam
     Azure::Nullable<DateTime> ExpiresOn;
     Azure::Nullable<DateTime> LastAccessedOn;
     bool IsDirectory = false;
-    Azure::Nullable<DataLakeArchiveStatus> ArchiveStatus;
+    Azure::Nullable<Models::ArchiveStatus> ArchiveStatus;
```

Comment:
Given the way the code is written (with using directives and common shared type names), this is a tricky bug to catch (`Models::ArchiveStatus` vs just `ArchiveStatus`). How did you catch it and are we confident we don't have any such issues elsewhere?

Reply:
code without `Models::` doesn't build with GCC and clang

Reply:
So if CI passes, we're good.